text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/internal/TagInternals.java

```diff
@@ -48,7 +49,7 @@ private TagInternals() {
    * @param tagName the name of the tag
    * @since 4.10.0
    */
-  public static void assertValidTagName(final @NotNull String tagName) {
+  public static void assertValidTagName(@org.intellij.lang.annotations.Pattern(TAG_NAME) final @NotNull String tagName) {
```

Comment:
Can we avoid this ugly import my making a meta annotation for this? I'm pretty sure the `Pattern` annotation supports being used on another annotation.

Reply:
Added a annotation in [`64e9571` (#920)](https://github.com/KyoriPowered/adventure/pull/920/commits/64e9571df11c29349274c6268ee953ba829f1340) although I'm not sure of the name and if it's in the right package